add ELAN 2BB3 (Asus Zenbook Flip S13 UX371)

vchlum opened the pull request:
Link to [wacom-hid-descriptors/issues/179](https://github.com/linuxwacom/wacom-hid-descriptors/issues/179)

Thank you!

whot commented on data/elan-9008.tablet:
```diff
@@ -0,0 +1,13 @@
+# ELAN touchscreen/pen sensor present in the ASUS ZenBook Flip S13 OLED UX371
```
```suggestion
# ELAN touchscreen/pen sensor present in the ASUS ZenBook Flip S13 OLED UX371
#
# sysinfo.rrI71nO76F
# https://github.com/linuxwacom/wacom-hid-descriptors/issues/179
```

whot commented:
test suite fails with this one:

> E           AssertionError: /home/runner/work/libwacom/libwacom/data/elan-9008.tablet: 04F3 must be lowercase hex

Should be an easy fix, please make sure to run `make check` or `ninja test` after the update in case there's another issue somewhere. saves on the turnaround time. Thanks

vchlum commented:
Fixed. I also changed the name to be consistent with the rest of the elan files. (ELAN 9008 was too general - I created the file manually because `git-update.sh` wasn't successful.)

I run the `make check` -> all 2 tests passed. I run also run `ninja test` -> all 7 tests passed.

whot commented:
Works now, thanks heaps!
